install-deps.sh

```diff
@@ -458,6 +458,7 @@ function preload_wheels_for_tox() {
     if test "$require" && ! test -d wheelhouse ; then
         type python3 > /dev/null 2>&1 || continue
         activate_virtualenv $top_srcdir || exit 1
+        python3 -m pip install --upgrade pip
```

Comment:
@ljflores  this command upgrade `pip` to the latest version available on pypi. while #43206 does practically the same thing -- to install a new enough pip in the activated virtualenv. the reason why i like it more is that it does this in the same place we install all the prerequisites for populating the wheels.

Reply:
I see, makes sense!

Reply:
@ljflores hi Laura, now that #43206 has been merged, could you please test the latest master or any commit which include the PR to see if it addresses the failure?

Reply:
Hey @tchaikov, this issue appears to be okay now. However, upon re-running install-deps to test for this issue, I ran into another. I left a comment on a commit of yours I believe is related. Let me know if you have any thoughts.

But as for this PR, I believe the issue has been solved.

https://github.com/ceph/ceph/commit/3185854c800fde1ed2e498d9fc692228f2db9277#commitcomment-56755869

Reply:
> Hey @tchaikov, this issue appears to be okay now. However, upon re-running install-deps to test for this issue, I ran into another. I left a comment on a commit of yours I believe is related. Let me know if you have any thoughts.
> 
> But as for this PR, I believe the issue has been solved.

thanks for testing this change. i am closing your PR then.

> 
> [3185854#commitcomment-56755869](https://github.com/ceph/ceph/commit/3185854c800fde1ed2e498d9fc692228f2db9277#commitcomment-56755869)

replied.